Should be removed once it will get into the mainline release.

Taken-from: http://bugs.exim.org/show_bug.cgi?id=1103
Reported-by: Oliver Brandmueller <ob@e-gitt.net>
--- src/exiqgrep.src.orig	2011-05-10 10:29:50.000000000 +0400
+++ src/exiqgrep.src	2011-05-10 10:31:13.000000000 +0400
@@ -88,7 +88,7 @@
 		chomp();
 		my $line = $_;
 		#Should be 1st line of record, if not error.
-		if ($line =~ /^\s*(\w+)\s+((?:\d+(?:\.\d+)?[A-Z])?)\s*(\w{6}-\w{6}-\w{2})\s+(<.*?>)/) {
+		if ($line =~ /^\s*(\w+)\s+((?:\d+(?:\.\d+)?[A-Z]?)?)\s*(\w{6}-\w{6}-\w{2})\s+(<.*?>)/) {
 			my $msg = $3;
 			$id{$msg}{age} = $1;
 			$id{$msg}{size} = $2;
